File: evidence/generated-systems/golang/fintech-on-golang/fintech-on-golang/internal/model/Branch.go
package model

import (
    "gorm.io/gorm"
)

//==============================================================
// Branch Declaration
//==============================================================
type Branch struct {
    gorm.Model
     Name                                    string
    BranchCode                                    string
    Address                                                            string
    InstitutionId         *uint
    Institution           *FinancialInstitution `gorm:"foreignKey:InstitutionId"`

// parent associations as their child

}

File: evidence/generated-systems/django/fintech-on-django/fintechOnDjango/models/Branch.py
from django.db import models

#======================================================================
# 
# Encapsulates data for model Branch
#
# @author Harbormaster Dev Team
#
#======================================================================

#======================================================================
# Class Branch Declaration
#======================================================================
class Branch (models.Model):

#======================================================================
# attribute declarations
#======================================================================
	name = models.CharField(max_length=200, null=True)
	branchCode = models.CharField(max_length=200, null=True)
	address = Address
	institution = models.ForeignKey('FinancialInstitution', on_delete=models.CASCADE, null=True, blank=True, related_name='+')

#======================================================================
# function declarations
#======================================================================
	def toString(self):
		str = ""
		str = str + self.name
		str = str + self.branchCode
		str = str + self.address
		return str;
    
	def __str__(self):
		return self.toString();

	def identity(self):
		return "Branch";
    
	def objectType(self):
		return "Branch";
